docs/reference/rest-api.rst

```diff
@@ -24,11 +24,11 @@ API.
  Reference
 ***********
 
-+----------------------------------------------------+
-| REST API Reference                                 |
-+====================================================+
-| `Determined REST API <../rest-api/index.html>`__   |
-+----------------------------------------------------+
++----------------------------------------------------------------------------------+
+| REST API Reference                                                               |
++==================================================================================+
+| `Determined REST API <https://docs.determined.ai/latest/rest-api/index.html>`_   |
```

Comment:
I don't think we needed to fix this one. It looks like it works properly on main.

Reply:
just to make sure it always works

Reply:
I'm not sure that's the case. This will always navigate to latest, but what if a user is looking at a specific version of the docs? So if we ship `0.28.3`, but the user wants to look at `0.28.1`, clicking on this link will take them to `0.28.3` unintentionally.

Reply:
I still consider this to be a bug, please fix.

Reply:
i see what's happening, it doesn't work when running from http://localhost:1234/reference/rest-api.html

Reply:
resolved